wave_matching/tests/icp_tests.cpp

```diff
@@ -0,0 +1,80 @@
+// Tests for wrapper
+
+#include <gtest/gtest.h>
+#include <pcl/io/pcd_io.h>
+#include <Eigen/Dense>
+#include <iostream>
+
+#include "wave/matching/ICP.hpp"
+
+bool isApprox(const Eigen::Affine3d& first,
+              const Eigen::Affine3d& second,
+              double TOL) {
+    for (uint8_t iii = 0; iii < 4; ++iii) {
+        for(uint8_t jjj = 0; jjj < 4; ++jjj) {
+            TOL -= std::abs(first(iii,jjj) - second(iii,jjj));
+        }
+    }
+    if (TOL > 0) {
+        return true;
+    }
+    return false;
+}
+
+// Fixture to load same pointcloud all the time
+class ICPTest : public testing::Test {
+  protected:
+    ICPTest() {}
+    virtual ~ICPTest() {}
+    virtual void SetUp() {
+        ref = boost::make_shared<pcl::PointCloud<pcl::PointXYZ> >();
+        target = boost::make_shared<pcl::PointCloud<pcl::PointXYZ> >();
+        pcl::io::loadPCDFile("../../wave_matching/tests/testscan.pcd", *ref);
+    }
+    void setparams(const float res, const Eigen::Affine3d perturb) {
```

Comment:
So, what is going on here? Can someone just suggest the correction.  setParams, right?